packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.styles.ts

```diff
@@ -16,11 +24,24 @@ export const getStyles = (props: IShimmeredDetailsListStyleProps): IShimmeredDet
             right: 0,
             bottom: 0,
             left: 0,
-            backgroundImage: `linear-gradient(to bottom, transparent 30%, ${palette.whiteTranslucent40} 65%,${palette.white} 100%)`
+            backgroundImage: `linear-gradient(
+              to bottom,
+              ${transparentColorStart} 30%,
+              ${transparentColorMiddle} 65%,
+              ${semanticColors.listBackground} 100%
+              )`
           }
         }
       },
       className
+    ],
+    rowPlaceholderWrapper: [
+      showCheckbox && {
+        marginLeft: 40 // 40px to take into account the checkbox of items if present.
```

Comment:
Would be great to share this value, but I imagine you explored that and it isn't easily do-able.

Reply:
I looked at this already and there was no obvious place to have it shared and pinned in one place, but I will take a quick look again and try to have the source in just one place.

Reply:
Looked again as it seemed to be very familiar to me and apparently I already did share this value in a PR for `fabric-7` branch:
https://github.com/OfficeDev/office-ui-fabric-react/blob/39cd8dcfb0a3e8fa6e5abbb4d6e4377b277bf33a/packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.styles.ts#L12
... so for now I am planning to leave it as is and with my shimmer cleanup in `DetailsRow` in fabric-7 branch after checking this changes I will make sure to have it using from that const.